src/GitHub.InlineReviews/Views/CommentView.xaml

```diff
@@ -118,7 +118,8 @@
                               Text="{Binding Body, UpdateSourceTrigger=PropertyChanged}"
                               TextWrapping="Wrap"
                               VerticalAlignment="Center"
-                              GotFocus="ReplyPlaceholder_GotFocus">
+                              GotFocus="ReplyPlaceholder_GotFocus"
+                              SpellCheck.IsEnabled="True">
```

Comment:
Wasn't there another PR with this feature?

Reply:
Well, there will be.  I didn't try to sneak this one in, seriusly! ;-)